test/txmeta-test.js

```diff
@@ -22,4 +22,13 @@ describe('TXMeta', function() {
     const txJSON2 = txmeta2.getJSON(network, null, 100);
     assert.strictEqual(txJSON2.confirmations, 1);
   });
+
+  it('should return blockhash as string', async () => {
+    // confirmed once at height 100
+    const block = Buffer.from('058f5cf9187d9f60729245956688022474ffc0eda80df6340a2053d5c4d149af');
+    const txmeta2 = TXMeta.fromOptions( {block} );
```

Comment:
Style nit: `({ block })` over `( {block} )`

Reply:
I really need to set my eslint back up.... Fixing this now haha